zerver/lib/integrations.py

```diff
@@ -4,9 +4,8 @@
 
 from django.contrib.staticfiles.storage import staticfiles_storage
 from django.urls import URLResolver, path
-from django.utils.functional import Promise
 from django.utils.module_loading import import_string
-from django.utils.translation import gettext as gettext_lazy
+from django.utils.translation import gettext
```

Comment:
Isn't the bug here that this should be using `gettext_lazy`, though?

Reply:
I see. Fixed.